examples/dae/disease_DAE.py

```diff
@@ -227,7 +227,7 @@ def _scaled_beta(model, i):
 model.con_city_varying_beta = Constraint(model.S_BETA, rule=_scaled_beta)
 
 def _mean_patt(model):
-    return (1.0, summation(model.beta_patt)/len(model.S_BETA))
+    return (1.0, sum_product(model.beta_patt)/len(model.S_BETA))
```

Comment:
This model is old and appeared to use `summation` instead of `sum` for no good reason.

Reply:
FWIW, `sum()` won't work, as it will sum over the indices of `model.beta_patt` and not the values.  You would need `sum(model.beta_patt[i] for i in model.beta_patt)`

Reply:
It's not clear whether a change is necessary.  Or perhaps the code should be changed back to summation()?